HipercowApi/Program.cs

```diff
@@ -19,19 +25,81 @@ public static void Main(string[] args)
         {
             var builder = WebApplication.CreateBuilder(args);
 
+            // Load JWT settings
+            builder.Services.Configure<JwtSettings>(builder.Configuration.GetSection("JwtSettings"));
+            var jwtSettings = builder.Configuration.GetSection("JwtSettings").Get<JwtSettings>()!;
```

Comment:
```suggestion
            var jwtSettings = builder.Configuration.GetSection("JwtSettings").Get<JwtSettings>()
    ?? throw new InvalidOperationException("JwtSettings section is missing or invalid in configuration.");
```

Dont have to put `!` after jwtSetting everywhere now